src/book/03-guides/03-hardware/04-DARwIn-OP2-guide.mdx

```diff
@@ -0,0 +1,184 @@
+---
+section: Guides
+chapter: Hardware
+title: DARwIn OP2 Robot Restoration and Calibration Guide
+description: How to use the provided instructions for setting up, restoring, and calibrating the DARwIn OP2 robot system.
+slug: /guides/hardware/DARwIn-OP2-guide
+authors:
+  - Corah Oliver (@yippyy)
+---
+
+## DARwIn OP2 Guide
+
+### DARwIN OP2 Specifications
+
+- Mainboard: PICO-CV01
+- Cpu: Intel Atom N2600 @1.6GHz single core
+- Ram: Up to 4GB DDR3/240-ping SO-DIMM module (user replaceable)
+- Storage: Half-size mSATA module (32GB) (user replaceable)
+- Sub controller: CM-740
+- Linux mint 17.3 'Rosa' installed on PICO-CV01 board
+
+### Creating the recovery USB
+
+1. Download Tuxboot from the [SourceForge website](https://sourceforge.net/projects/tuxboot/files/0.6/). For this demonstration we will be downloading the Linux version.
+   ![Downloading Tuxboot 1](./images/DARwIn_OP2_guide/Tuxboot1.jpg)
```

Comment:
The alt text here isn't very useful. Something like "the SourceForge Tuxboot page showing the Home/0.6 directory". A caption could be useful too, "like navigate into the 0.6 folder".